File: PHP/XadrezTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("Xadrez.php");

class XadrezTest extends TestCase {

    public function testXadrez() {
        $obj = new Xadrez();
        //Test 1
        $want = 1;
        $got = $obj->tableColor(8, 8);
        $this->assertEquals($want, $got);
        //Test 2
        $want = 1;
        $got = $obj->tableColor(5, 91);
        $this->assertEquals($want, $got);
        //Test 3
        $want = 0;
        $got = $obj->tableColor(6, 9);
        $this->assertEquals($want, $got);
    }

}

File: GO/Xadrez_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestXadrez(t *testing.T) {
	tables := []struct {
		L    int
		C    int
		want int
	}{
		{8, 8, 1},
		{5, 91, 1},
		{6, 9, 0},
	}

	for _, table := range tables {
		got := tableColor(table.L, table.C)

		if got != table.want {
			t.Errorf("Waiting for this %d but the return was this: %d", table.want, got)
		}
	}
}
